ro_help/hub/migrations/0049_models_to_city_model.py

```diff
@@ -0,0 +1,119 @@
+# Custom migration
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+def map_city_field_to_city_model(apps, schema_editor):
+    print(f"\n{'*'*80}\n")
+    print("[INFO]: START of data migration")
+
+    City = apps.get_model("hub", "City")
+
+    NGO = apps.get_model("hub", "NGO")
+    NGONeed = apps.get_model("hub", "NGONeed")
+    PersonalRequest = apps.get_model("hub", "PersonalRequest")
+    RegisterNGORequest = apps.get_model("hub", "RegisterNGORequest")
+
+    models = [NGO, NGONeed, PersonalRequest, RegisterNGORequest]
+
+    for model in models:
+        for n in model.objects.all():
+            try:
+                if "sector" in n.county.lower():
+                    n.county = "BUCURESTI"
+
+                city = n.city.lower().strip()
+                # remove romanian characters
```

Comment:
The main city source that we used from our partners do not have Romanian characters, so we are trying to replace the existing ones and match as many as possible automatically.